Makefile

```diff
@@ -644,7 +644,7 @@ docs-linkcheck: /usr/local/bin/markdown-link-check
 	markdown-link-check -q -c .mlc_config.json $(shell find docs -name '*.md' -not -name fields.md -not -name swagger.md -not -name executor_swagger.md)
 
 /usr/local/bin/markdownlint:
-	npm i -g  markdownlint-cli # update this in Nix when upgrading it here
+	npm i -g markdownlint-cli@0.33.0 # update this in Nix when upgrading it here
```

Comment:
Can you also up this in Nix (dev folder)?

Reply:
Per the PR description, this _is_ the version used in the Nixfiles

Reply:
Unless you meant updating all of the above to [latest](https://github.com/igorshubovych/markdownlint-cli/releases/tag/v0.35.0)?

Reply:
Resolved discussion as this PR was approved, so I am assuming this good to go now. Just waiting on merge